core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -728,6 +746,11 @@ private[spark] object Utils extends Logging {
     localDirs
   }
 
+  /** Used by unit tests. Do not call from other places. */
+  private[spark] def clearLocalRootDirs(): Unit = {
```

Comment:
Note this method doesn't delete anything. There's a separate PR to clean up the root directories (#4759).